Makefile

```diff
@@ -0,0 +1,232 @@
+BASE_PACKAGES := utils db settings visualization
+
+MODELS := biocro clm45 dalec ed gday jules linkages \
+	      lpjguess maat maespa preles sipnet
+
+
+MODULES := allometry assim.batch assim.sequential benchmark \
+	       data.atmosphere data.hydrology data.land \
+	       data.mining data.remote emulator meta.analysis \
+	       photosynthesis priors rtm uncertainty
+
+.PHONY: all ${BASE_PACKAGES} ${MODELS} ${MODULES}
+
+all: .install PEcAn.all ${BASE_PACKAGES} ${MODELS} ${MODULES}
```

Comment:
Another option that can be a target is tests which would do the integration tests we have. You should be able to copy the whole block of code from build.sh. This way we just end up with a single file we need to maintain, and as a bonus we might be able to run multiple tests in parallel. Just need to make sure each test writes to its own folder.